CHANGELOG.md

```diff
@@ -1,8 +1,22 @@
 # Changelog
 
 ## Unreleased
+- Upgrade Node.js to 7.4.x
+([#63](https://github.com/fs/react-base/pull/63))
```

Comment:
Maybe it'd be more readable if we organize it like this:
[#63](https://github.com/fs/react-base/pull/63)
- Upgrade Node.js to 7.4.x
- Add transform-react-constant-elements, transform-react-inline-elements babel plugins
- so on